go.mod

```diff
@@ -6,3 +6,5 @@ require (
 	github.com/hashicorp/go-immutable-radix v1.3.0
 	github.com/hashicorp/golang-lru v0.5.4 // indirect
 )
+
+replace github.com/hashicorp/go-immutable-radix => github.com/hashicorp/go-immutable-radix v1.3.1-0.20210121185740-67e10d480dcf
```

Comment:
Leaving a note as a reminder that this should be changed to an official release, before merging this PR, if we are interested in this approach.